src/Raven.Server/Commercial/SetupManager.cs

```diff
@@ -772,8 +773,23 @@ public static IEnumerable<string> GetCertificateAlternativeNames(X509Certificate
 
                             var publicServerUrl =
                                 GetServerUrlFromCertificate(serverCert, setupInfo, LocalNodeTag, setupInfo.NodeSetupInfos[LocalNodeTag].Port, out var domain);
+
+                            setupInfo.Domain = domain;
```

Comment:
This was intentionally removed. We need this to be the _global_ domain, for all the nodes, not `a.domain`, etc.

Reply:
We still need this for secured setup where we don't get a domain from the user.